fact_extractor/plugins/unpacking/patool/test/data/test.a

```diff
@@ -0,0 +1,3 @@
+!<arch>
+test.data/      0           0     0     644     128       `
+d34bb4669ac6e2b15b07f2489a22c289974a3dcfddeec68fc003d0109186e34dbc4c419546211e534c4be6ae657397959ae8b4a5e2c095c5bba3e0bca38f0b51
```

Comment:
Is the missing newline intendet?

Reply:
This isn't actually a text file, it is a file packed with `ar`. I don't know why git detects it as text file. I tried to add a `.gitattributes` file as "*.a binary" but it didn't seem to work and was still detected as text.